src/PactNet/Interop/StringResult.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Runtime.InteropServices;
+namespace PactNet.Interop
+{
+    /// <summary>
+    /// Represents the result of a string operation.

```

Comment:
My understanding is that this enum stores a "result" of an operation (not necessarily a string operation). The result is either a success, and there are potential messages associated with the OK state or Failed state.

@uglyog could you please clarify?

Reply:
That comment is wrong, it can be any operation. Also, I don't think the .Net layer need be concerned with the success value, only if there is an error, that needs to be presented to the user.

Reply:
Thanks. I couldn't see when the success string might be populated.